src/GraphList.hpp

```diff
@@ -50,6 +50,14 @@ template <typename VertexType, typename EdgeType> class GraphList {
       Exceptions::handle_exception_map(resp);
   }
 
+  // Helper method to call updateEdge method from PeakStore
+  template <typename E = EdgeType>
+  auto updateEdge(const VertexType &src, const VertexType &dest,
+               const EdgeType &newWeight)
+    -> std::enable_if_t<!CinderPeak::Traits::is_unweighted_v<E>, PeakStatus> {
```

Comment:
```suggestion
    -> std::enable_if_t<!CinderPeak::Traits::is_unweighted_v<E>, bool> {
```
I am not fully sure if we should return/expose ``PeakStatus`` to the user, it's supposed to be used internally, maybe we can return a ``bool`` or something which can be a better alternative.

Reply:
@SharonIV0x86 Yeah we can return bool indicating whether the edge updation was successful or not.

Should I proceed with this?

Reply:
Yeah go ahead.